### ISSUE TITLE
Sourceforge website contains outdated information

### ISSUE THREAD

**Issue by RalfJung** [OpenMandrivaSoftware/ossp#6]
The old sourceforge website https://sourceforge.net/projects/osspd/ contains no indication that it is outdated. I am the Debian maintainer of osspd, and I had no idea that it is outdated. Someone updated https://sourceforge.net/projects/fuse so I hope that someone also has the access to fix the osspd page. :)

It'd be good to add a note to the website, and maybe also send an announcement to the [mailing list](https://sourceforge.net/p/osspd/mailman/osspd-users/) that git remotes need to be updated.

**Comment by davidebeatrici** [comment]
I would also lock/disable the repository on SourceForge, to prevent it from diverging (again) from this one.

**Comment by Nikratio** [comment]
I'm afraid this project is completely abandoned, both on Sourceforge and on Github.

If someone tells me how to disable the SourceForge project then I can try to do so. I just tried but didn't find a way to do that.

**Comment by RalfJung** [comment]
I never saw the sourceforce project admin UI, so I cannot help with that. Is there a way to change the project description? Then you could make that a reference to this repository.

**Comment by Nikratio** [comment]
I can't even find an Admin UI for the project

**Comment by davidebeatrici** [comment]
In theory you can find it at the top right of the page, if logged in.

**Comment by davidebeatrici** [comment]
If you're not interested in maintaining this project anymore, @OpenMandrivaSoftware will gladly take over it.

While (I assume) it was originally created to aid the transition from OSSv3 to ALSA, I believe if we implement OSSv4 support it will become relevant again.

Right now the only way to develop for that sound system on Linux (without jeopardizing ALSA) would be to run a *BSD in a virtual machine.

**Comment by Nikratio** [comment]
Happy to hand it over, just tell me exactly what I need to do where.

**Comment by davidebeatrici** [comment]
Awesome! The relevant settings can be found at https://github.com/libfuse/osspd/settings.

![](https://user-images.githubusercontent.com/5897523/185987754-807d037b-d727-495b-a231-778896bda765.png)

![](https://user-images.githubusercontent.com/5897523/185988047-8c3bfba0-a183-4402-89bd-1d78f1949724.png)

**Comment by Nikratio** [comment]
Yes, I know what to do on GitHub. My question was about the SourceForge project :-).

**Comment by Nikratio** [comment]
Also, for the GitHub transfer: 
![image](https://user-images.githubusercontent.com/701625/186119238-f8e6f5e5-2ed6-4f13-9b25-dca25b98f8a7.png)

**Comment by Nikratio** [comment]
I've added you as a GitHub collaborator for now.

**Comment by berolinux** [comment]
I've sent you an invite that should make transferring the repository possible.

**Comment by davidebeatrici** [comment]
Thanks, for reference: https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository

> Prerequisites for repository transfers:
>
> - When you transfer a repository that you own to another personal account, the new owner will receive a confirmation email. The confirmation email includes instructions for accepting the transfer. If the new owner doesn't accept the transfer within one day, the invitation will expire.
> - To transfer a repository that you own to an organization, you must have permission to create a repository in the target organization.

I expected GitHub to send a confirmation email to @OpenMandrivaSoftware's owner(s) instead.

**Comment by berolinux** [comment]
On sourceforge, go to
https://sourceforge.net/p/osspd/admin/overview
There, on the right, you can say "Moved Project to:" and replace it with a pointer to the new URL.

**Comment by Nikratio** [comment]
That just gives me a "Forbidden" error, sorry. (Yes, I am logged in).

**Comment by davidebeatrici** [comment]
![](https://user-images.githubusercontent.com/5897523/186524795-fed554ff-99dd-4b89-a49c-db82df5f93b7.png)

@htejun @Lantizia

**Comment by Nikratio** [comment]
Could you please remove me from the OpenMandriva team again?

**Comment by berolinux** [comment]
done

**Comment by Lantizia** [comment]
I wasn't ever really that integral to this project... Tejun mostly only listed me as I think he didn't have the time to maintain it any more.

I think the last time I was properly involved was around 2012 and a bunch of us (mostly with myself rallying everyone else) were trying to get a bunch of patches included in the kernel to fix something underneath to allow osspd to work better...

This thread has all 3 patches and a few thoughts from people (including Linus) after...

https://lkml.org/lkml/2012/1/13/211

The only other thing I can find is me moaning about the kernel that shipped with Ubuntu 12.04 being compiled to reserve /dev/dsp (despite the fact Ubuntu had removed OSS and OSS backwards compatibility long ago by that point) as this stopped osspd assuming it, making it impossible to use.

https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1105230

There is a plethora of commercial (and thus no source code) games that were sold and published in the late 90's/early 2000's (such as Loki ported games) which look for OSS and can still work on a modern Linux distribution with osspd (once you get around the headache of older GTK libraries and such).

Ultimately I still care very deeply about osspd and I'll do any testing that people might need of me... and contribute in any other way that I am capable of helping.  But from my perspective the interest is only in emulating older OSS versions from that late 90's / early 2000's era.

I do remember some of these games weren't as "fooled" as others and either didn't work, or didn't work as well (but most worked fine)... and I think it was the above kernel patches which was meant to address it (but I'm not sure if any of them got added).

I have a little collection of some of these kinds of games (in their original big box form)... if anyone ever needs me to do some testing.

**Comment by davidebeatrici** [comment]
Awesome, thank you very much for your valuable input!

Please let me know whether you would like to be added to this repository.

Do you perhaps have the privileges to modify the project on SourceForge?

**Comment by Lantizia** [comment]
It would appear I have admin access yes... what would you want modifying and why?

If it is to mark this repository is the legitimate successor then I'd feel a little uneasy about doing that without Tejun's input **and/or** some overwhelming indication of why this repository ought to be that successor.  

Notice the 'and/or' used there... whilst this might have always been Tejun's thing... I've not heard from him in a long time and I'm not entirely sure he's all that fussed either way... so it may be a long shot expecting any input at this point.

I'll try highlighting him again though... oh @htejun :)

Personally I'd be wondering why **this** repository and why use a repository that is linked/associated with... a specific distribution like OpenMandriva?

**Comment by htejun** [comment]
Hey, I haven't done anything w/ osspd for like a decade. Given that this is the first time I hear about it in such a long time, if there are people with real use cases and interest in maintaining the project, it makes perfect sense to transfer the maintainership. @Lantizia, if you have can easily do the sourceforge side, please go ahead with whatever is necessary. Thanks.
